doc/pick_place/src/pick_place_tutorial.cpp

```diff
@@ -44,6 +44,9 @@
 // TF2
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
 
+// The circle constant tau = 2*pi. One tau is one rotation in radians.
+const double tau = 6.283185307179586476925;
```

Comment:
Python apparently defines it as 2*pi, why shouldn't we use `const double tau = 2 * M_PI`?

Reply:
I like that better, too. Changed.